src/lib/x509/x509_obj.h

```diff
@@ -47,11 +47,6 @@ class BOTAN_PUBLIC_API(2,0) X509_Object : public ASN1_Object
       */
       const AlgorithmIdentifier& signature_algorithm() const { return m_sig_algo; }
 
-      /**
-      * @return hash algorithm that was used to generate signature
-      */
-      std::string hash_used_for_signature() const;
```

Comment:
For my own understanding: when is an existing `BOTAN_PUBLIC_API(major,minor)` supposed to get changed?

- Then the API is extended? (probably not)
- Then the API is broken? E.g.
  - By removing a part of the API
  - Or changing a part of the API
- Never, as it is just supposed to indicated when some version of the API was added, regardless if it is still compatibel with it.